pkg/utils/utils.go

```diff
@@ -7,6 +7,20 @@ import (
 	"github.com/openshift/origin/third_party/github.com/docker/libcompose/project"
 )
 
+// CheckIfFileExists checks if all files in array exists and that they are files (not directories).
+func CheckIfFileExists(files []string) error {
+	for _, filename := range files {
+		fileInfo, err := os.Stat(filename)
+		if err != nil {
+			return fmt.Errorf("main: file %q not found", filename)
```

Comment:
ok, than I understand. 
And my question is why would you  add "main: "? 
I guess this is because it comes from main function, right?

Why should user see that? He doesn't care about that.

If you want it there, I would put it as separate debug message.